psl-core/src/main/java/org/linqs/psl/config/Options.java

```diff
@@ -303,6 +279,13 @@
         null
     );
 
+    public static final Option WLA_GPP_USE_PROVIDED_WEIGHT = new Option(
+            "gpp.useProvidedWeight",
```

Comment:
Too many indentations.

Remember to always match the style in the file.
(Although the standard rule is to indent twice here, since almost the entire file is made up of these definitions, we only indent once.)